lib/connection.js

```diff
@@ -357,11 +396,15 @@ function postgresSocket(options, {
       succeeded = true
       i >= options.host.length && (i = 0)
     },
-    write: async x => {
+    pause: () => socket.pause(),
+    resume: () => socket.resume(),
+    isPaused: () => socket.isPaused(),
```

Comment:
`pause`, `resume` and `isPaused` doesn't exists on `socket` (which is `Deno.Conn`); the `copy` feature won't work at all. I think you should disable the copy feature since it's a bit out of scope here.

If you really want to re-implement the pause mechanism of Node sockets, you should take a look at the Deno read loop:
https://github.com/Minigugus/postgres-deno/blob/b84f78cb0caf46f6ed45984be911b67b7a14fa1d/lib/deno.js#L28-L34
The loop has to `await` on some kind of lock that then implements `pause()` and `resume()`